asv/benchmarks/dwt_benchmarks.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import pywt
```

Comment:
I think for newer functions the way to write the import in asv is:

    try:
        from pywt import dwt, idwt
    except ImportError:
        pass

Otherwise when you run the benchmarks for newly added functions against an older ``pywt`` version that doesn't have them, the tests crash. Obviously ``dwt, idwt`` have existed for a while, but if we're going to copy this as a template for other benchmarks, maybe good to keep the style uniform.

See https://github.com/scipy/scipy/blob/master/benchmarks/README.rst#writing-benchmarks for this and other benchmarking pointers to steal for a basic README here.